src/IRrecv.h

```diff
@@ -18,6 +18,7 @@
 #define HEADER         2U  // Usual nr. of header entries.
 #define FOOTER         2U  // Usual nr. of footer (stop bits) entries.
 #define OFFSET_START   1U  // Usual rawbuf entry to start processing from.
+#define MS_TO_USEC  1000U  // Convert milli-Seconds to micro-Seconds.
```

Comment:
I suggest here

#define MS_TO_USEC(x) (x *1000)

Then usage is

MS_TO_USEC(irparams.timeout)